app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
@differentreality : in this case, should I ask `if can? :manage, Conference || can? :manage, CallForPaper || can? :manage, Registration || can? :manage, Vposition` ?? because after I modified `signed_in_with_roles(user)` in ability.rb, the link is not visible with `if can? :manage, Conference` condition